azure-tests/src/main/java/fixtures/azurereport/AutoRestReportServiceForAzure.java

```diff
@@ -45,7 +45,7 @@ public String getHost() {
      * @param host the host value.
      * @return the service client itself.
      */
-    public AutoRestReportServiceForAzure setHost(String host) {
+    AutoRestReportServiceForAzure setHost(String host) {
```

Comment:
Make it package private for now (if builder and client is in same package).

Later will move it to constructor, so that client will be immutable.